yarn.lock

```diff
@@ -0,0 +1,14215 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
+"@apollo/react-common@^3.1.4":
```

Comment:
giant yarn.lock file added, did you try using yarn? this is an npm project so delete this since we already have a package-lock.kson